android/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,7 @@
     package="com.linagora.android.tmail">
 
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="28"/>
```

Comment:
@huynguyennovem when we use: `flutter_downloader: 1.7.0` do we need to set `android:maxSdkVersion="28"`?

Reply:
we should add this to ignore the warning by the WRITE permission will no longer provided when targeting Android 10 and above

Reply:
ah, you mean, because we target to Android 10 and above, so we need to set this flag to avoid warning (this still need for Android before 10).

Reply:
Right. It will need for Android 9 and below. From Android 10 and above, it does not need

Reply:
Right. The flag means that the permission will need for Android 9 and below. From Android 10 and above, it does not need